torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4792,6 +4792,9 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            sample_inputs_func=sample_inputs_pow,
            supports_inplace_autograd=False,
            assert_autodiffed=True),
+    OpInfo('float_power',
+           dtypes=all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool),
+           sample_inputs_func=sample_inputs_pow),
```

Comment:
minor:
maybe update the name `sample_inputs_pow` to `sample_inputs_pow_float_power` to reflect it provides samples for both of them
or
maybe not as it is not surprising to see it being used for `float_power` as well.

@mruberry what do you feel?

Reply:
It's a reasonable nit but let's stick with this for now.